census/vanilla-tf/trainer/model.py

```diff
@@ -46,22 +30,27 @@ def inference(input_x, hidden_units=[100,70,50,25], num_classes=2):
     num_classes (int): Number of classes
 
   Returns:
-    Feed forward NN with relu and softmax layer
+    The last layer values
   """
-  layers_size = [input_x.get_shape()[1]] + hidden_units
-  layers_shape = zip(layers_size[0:],layers_size[1:])
+  print(inputs.get_shape())
+  layers_size = [inputs.get_shape()[1]] + hidden_units
+  layers_shape = zip(layers_size[0:],layers_size[1:] + [num_classes])
+
+  curr_layer = inputs
+  with tf.variable_scope('dnn',
```

Comment:
If we want to demonstrate output, we can do the softmax from the logit layers here and return `logits, predictions` as a tuple. But doing a softmax layer before the logits is just wrong.